### ISSUE TITLE
Condition for Terminating a Communication Session

### ISSUE THREAD

**Issue by UweSteinkeFromSiemens** [openETCS/modeling#907]
Subset 026, 3.5.5.1 f): 

"The termination of a communication session shall be initiated ... in the following cases: ... 

f) In case of a communication session with an RIU: the Level 1 is left, or, an End of Mission is executed."

My interpretation is, that this case applies " In case of a communication session with an RIU". End of mission causes the termination of the current session only, when it is established with a RIU, but not with an RBC. 

Is this, what is intended?

**Comment by ZeHiro** [comment]
Yes. The RBC is in charge of closing the connection when the onboard goes to L1

**Comment by ZeHiro** [comment]
The RBC is responsible to close the communication in this case. See §5.5.3.1.3
